pikaur/install_info_fetcher.py

```diff
@@ -528,9 +528,10 @@ def get_aur_pkgs_info(  # pylint: disable=too-many-branches
                 self.package_is_ignored(pkg_name)
             ):
                 del aur_updates_install_info_by_name[pkg_name]
-            for pkg_list in self.pkgbuilds_packagelists.values():
-                if pkg_name in pkg_list:
-                    del aur_updates_install_info_by_name[pkg_name]
```

Comment:
just check if it's in this dict or not, instead of changing the logical branching

Reply:
But the nested loop will just do nothing in case the key already got removed ? Do you plan to add logic there? Or is it possible that pair isn't in the dict to begin with and we should check at both places? There is no other place in the code where pairs are removed from this dict.

Reply:
apparently it could get added more than once during interactive user interaction (which currently not covered by the tests, since it's hard to implement):

f9d7e64d584a760801de75ec4c36aff1503e15b3

also commit message seems to describe a similar situation to your original issue

Reply:
ah, my bad, it's a dict